client/stylesheets/_toCat.scss

```diff
@@ -4616,21 +4623,23 @@ On click the images will be displayed at normal size to complete the effect
 
 .searchResultsCards {
     vertical-align: top !important;
-    margin-top: -965px !important;
+    margin-top: -1300px !important;
     float: right;
 }
 
 .searchResultsCardUl > li {
     display: inline-block;
     width: 85%;
-    padding: 10px;
-    border: 2px solid #00bfff;
+    padding: 13px 10px 10px 10px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Shorthand form for property `padding` should be written more concisely as `13px 10px 10px` instead of `13px 10px 10px 10px`